source/command-line-options.txt

```diff
@@ -33,18 +35,77 @@ Use the following syntax to start |compass| and set a configuration option:
 Options
 -------
 
-You can set the following configuration options using the command line:
+You can set these options when using the command line:
+
+.. option:: --autoUpdates
+  
+   Enable automatic updates.
+  
+.. option:: --enableFeedbackPanel
+  
+   Enable the feedback panel.
+   
+.. option:: --enableMaps
+ 
+   Enable geographic visualizations.
+
+.. option:: --exportConnections
+    
+   Export :ref:`favorite connections 
+   <compass-export-import-cli-landing>`.
+
+.. option:: --file <file>
+  
+   Specify a file containing a list of connections. These connections
+   are automatically available once |compass| starts.
+
+.. option:: --help
+   
+   Show |compass| options.
+
+.. option:: --importConnections
+
+   Import :ref:`favorite connections 
+   <compass-export-import-cli-landing>`.
+
+.. option:: --lgDarkmode
+
+   Use leafygreen dark mode instead of darkreader.
+
+.. option:: --networkTraffic
+
+   Configure |compass| to not perform outgoing 
+   :ref:`network operations <compass-configure-network-traffic>`
+   other than those to the database.
+
+.. option:: --passphrase
+ 
+   Connection export/import passphrase.
```

Comment:
Rather than the /, consider something like:

The passphrase that protects the :link:`exported` connection credentials.

Reply:
Updated. Thanks!